.github/workflows/composite/test-sdk/action.yml

```diff
@@ -0,0 +1,30 @@
+name: 'Test SDK'
+description: 'Test SDK using a given test file.'
+inputs:
+  test-file: 
+    description: 'Name of the test file to run.'
+    required: true
+  test-report-dir:
+    description: 'Location of the test reports.'
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Install ruby
+      shell: bash
+      run: sudo apt-get install ruby-full
```

Comment:
I opted to commenting the action out as a compromise - since in the workflow the testing jobs are being commented out as well. Reasoning is that it serves as a template for what the test action should look like, to avoid having to go to other SDKs when we enable testing.

Reply:
Seeing that these were removed and will be reintroduced correctly in the upcoming component tests [PR ](https://github.com/phoenixnap/go-sdk-bmc/pull/40)